gio/src/subclass/application.rs

```diff
@@ -468,7 +468,5 @@ mod tests {
             .build();
 
         app.set_inactivity_timeout(10000);
-
-        assert_eq!(app.run_with_args(&["--local"]), EXIT_STATUS);
```

Comment:
This can be done again now or not?

But you probably want to switch `local_command_line()`, `command_line()` and `handle_local_options()` to also use the new type here

Reply:
Done, thanks I forgot about it since it wasn't possible without this wrapper.

Reply:
You missed `handle_local_options()`